<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0,0,0,0">
         <property name="sizeConstraint">
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <property name="horizontalSpacing">
          <number>3</number>
         </property>
         <property name="verticalSpacing">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="3" column="0">
          <widget class="QPushButton" name="num7Button">
           <property name="text">
            <string>7</string>
           </property>
           <property name="shortcut">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="num5Button">
           <property name="text">
            <string>5</string>
           </property>
           <property name="shortcut">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="num8Button">
           <property name="text">
            <string>8</string>
           </property>
           <property name="shortcut">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="num2Button">
           <property name="text">
            <string>2</string>
           </property>
           <property name="shortcut">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="num1Button">
           <property name="text">
            <string>1</string>
           </property>
           <property name="shortcut">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QPushButton" name="num0Button">
           <property name="text">
            <string>0</string>
           </property>
           <property name="shortcut">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0" colspan="2">
          <widget class="QPushButton" name="spaceButton">
           <property name="text">
            <string>Space</string>
           </property>
           <property name="shortcut">
            <string>Space</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="4">
          <widget class="QLineEdit" name="inputLine">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="num4Button">
           <property name="text">
            <string>4</string>
           </property>
           <property name="shortcut">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="8" column="3" rowspan="2">
          <widget class="QPushButton" name="enterButton">
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Enter</string>
           </property>
           <property name="shortcut">
            <string>Return</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QPushButton" name="num3Button">
           <property name="text">
            <string>3</string>
           </property>
           <property name="shortcut">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QPushButton" name="num6Button">
           <property name="text">
            <string>6</string>
           </property>
           <property name="shortcut">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QPushButton" name="num9Button">
           <property name="text">
            <string>9</string>
           </property>
           <property name="shortcut">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QPushButton" name="pointButton">
           <property name="text">
            <string>.</string>
           </property>
           <property name="shortcut">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item row="5" column="3" rowspan="2">
          <widget class="QPushButton" name="plusButton">
           <property name="text">
            <string>+</string>
           </property>
           <property name="shortcut">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QPushButton" name="supprButton">
           <property name="text">
            <string>Del</string>
           </property>
           <property name="shortcut">
            <string>Backspace</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="clearButton">
           <property name="text">
            <string>Suppr</string>
           </property>
           <property name="shortcut">
            <string>Del</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="divButton">
           <property name="text">
            <string>/</string>
           </property>
           <property name="shortcut">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="mulButton">
           <property name="text">
            <string>*</string>
           </property>
           <property name="shortcut">
            <string>*</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3" rowspan="2">
          <widget class="QPushButton" name="lessButton">
           <property name="text">
            <string>-</string>
           </property>
           <property name="shortcut">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QListView" name="listPile"/>
         </item>
         <item row="2" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>455</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOption"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
